src/pages/tutorials/aiops/installation/index.mdx

```diff
@@ -70,36 +69,62 @@ You can check your entitlement for Watson AIOPs by clicking **View Library**
+**NOTE:** If your entitlement key is not valid for Watson for AIOps Event Manager, you can register for a 60 day trial using the following link: [Request Watson AIOps trial](https://www.ibm.com/account/reg/us-en/login?formid=urx-51074).
 
 ## 2-4: Prepare your OpenShift cluster
 
-This step involves creating a namespace to install the Watson for AIOps Event Manager Operator into, and setting up your software entitlement key, so that you will be able to deploy an instance of Event Manager.
+This step involves creating a namespace (also known as a "Project" on OpenShift) to install the Watson for AIOps Event Manager Operator into, and setting up your software entitlement key, so that you will be able to deploy an instance of Event Manager.
 
-This step involves the following tasks:
+This section involves the following tasks:
 
-- Open the **OpenShift UI**
-- Acquire a token and log in to the OpenShift cluster via the command line
+- Navigate to and open the **OpenShift UI**
+- Establish a connection to your control virtual machine using the supplied SSH key
+- Acquire an OpenShift login token and log in to the OpenShift cluster via the command line
 - Create a **namespace** to deploy Event Manager into
 - Create an **entitlement key secret**
 
-Go to your newly created OpenShift cluster and click OpenShift web console to open the main OpenShift UI:
+During this module, you will need to log in to your dedicated "control" virtual machine, to run configuration commands against your OpenShift cluster.
+
+Prior to starting this lab, you should have received two emails: one containing the IP address of your "control" virtual machine, and one containing a zip file with an id_rsa private and public key, which you will need to access the control VM.
+
+- Follow the instructions in the email to unzip the id_rsa.zip file using the supplied password and save the key files onto your laptop
+- Open a command terminal (on Windows, use "Windows Powershell")
+- In the terminal session, navigate to the directory where your rsa_id file is located
+- Open an SSH session as the "jammer" user to your control box using the IP address provided to you in the other email - example:
+
+```sh
+ssh -i rsa_id jammer@10.20.30.40
+```
+
+After you have successfully opened a terminal session to your control virtual machine, switch to your Internet browser and log in to [https://cloud.ibm.com](https://cloud.ibm.com), and select the account labelled **2576490 - itztsgltechlabs01** from the top-right drop-down menu. Next, click on the menu button in the top-left corner, select **OpenShift**, then **Clusters**.
+
+You should see two OpenShift clusters listed: one labelled "manage-from" and one "manage-to". For the Event Manager sections, we will be working with the "manage-from" cluster. Click on the "manage-from" OpenShift cluster, then click the blue **OpenShift web console** button to open the main OpenShift UI:
```

Comment:
We should reword this to simply call out the user should perform these actions on their manage-from cluster. Outside scheduled tech jam events, users will need to provision their own infrastructure and clusters.